src/utils/translateWhiskyNameToJapenese.ts

```
@@ -0,0 +1,72 @@
+    /(\d+)\s*yo/i, // 12yo, 12 YO
+    /(\d+)\s*y/i, // 12y, 12 Y
+  ];
+
+  for (const pattern of yearPatterns) {
+    const match = input.match(pattern);
+    if (match && match[1]) return match[1]; // 숫자 그룹만 반환
+  }
+
+  // 숫자만 있는 경우 (예: 12)
+  const numberMatch = input.match(/\d+/);
+  if (numberMatch) return numberMatch[0];
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)